ui/apps/platform/src/Containers/Policies/Wizard/Step1/PolicyCategoriesSelectField.tsx

```diff
@@ -39,23 +74,105 @@ function PolicyCategoriesSelectField(): ReactElement {
             .catch(() => {});
     }, []);
 
+    // Filter available options based on input and already selected items
+    const filteredOptions = useMemo(
```

Comment:
This doesn't seem too expensive to me, and reruns every time the input changes anyway. Thoughts on simplifying and removing `useMemo`?